earth_enterprise/src/server/gepgcommon

```diff
@@ -0,0 +1,96 @@
+#! /bin/bash
+#
+# Copyright 2017 Open GEE Developers.
```

Comment:
update the year to 2018 ? may want to do incrementally in other places as we touch them.

Reply:
You, generally, use the earliest year something was copyrighted.  Alternatively, you can explicitly include all years for which you have a copyright, in case it may be transferred.  We can go either way, although I don't enjoy updating all file comments every year.

Reply:
I will update the year in the files as I make changes